assessments/guide/outline.md

```diff
@@ -1,18 +1,13 @@
 # Outline
 
-First of all, the burden is primarily on the proposing project to
-demonstrate it is secure in a manner that is understandable to the broader
-community.  The [reviewers](security-reviewer.md) will help to assess and probe the design.
+     The burden is primarily on the proposing project to demonstrate it is secure in a manner that is understandable to the broader community.  The [reviewers](security-reviewer.md) will help to assess and probe the design.
 
```

Comment:
leading space causing strange formatting...

![image](https://user-images.githubusercontent.com/41906/59937916-7bd95280-9408-11e9-9e61-de51350b1a63.png)

Reply:
i think i have this fixed